rewrite-yaml/build.gradle.kts

```diff
@@ -1,8 +1,22 @@
+// run manually with -x compileKotlin when you need to regenerate
+tasks.register<JavaExec>("generateAntlrSources") {
+    main = "org.antlr.v4.Tool"
+
+    args = listOf(
+        "-o", "src/main/java/org/openrewrite/yaml/internal/grammar",
+        "-package", "org.openrewrite.yaml.internal.grammar",
+        "-visitor"
+    ) + fileTree("src/main/antlr").matching { include("**/*.g4") }.map { it.path }
+
+    classpath = sourceSets["main"].runtimeClasspath
+}
+
 dependencies {
     api(project(":rewrite-core"))
     api("org.jetbrains:annotations:latest.release")
     api("com.fasterxml.jackson.core:jackson-annotations:2.12.+")
 
+    implementation("org.antlr:antlr4:4.8-1")

```

Comment:
You will want to move this to `org.antlr:antlr4:4.9.2`

Reply:
oh and then generate the source again.

Reply:
👍 ✅